include/robot_fingers/trifinger_platform.hpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * @file
+ * @brief TODO
+ * @copyright 2020, Max Planck Gesellschaft.  All rights reserved.
+ * @license BSD 3-clause
+ */
+#pragma once
+
+// TODO move to a separate package to not add unnecessary dependencies to
+// robot_fingers?
+#include <robot_interfaces/finger_types.hpp>
+#include <trifinger_object_tracking/object_tracker_data.hpp>
+
+namespace robot_fingers
+{
+// TODO move implementation to cpp file
+// NOTE: Inheriting from Frontend makes implementation easier but means that we
+// cannot rename get_observation
+class TriFingerPlattform : public robot_interfaces::TriFingerTypes::Frontend
+{
+public:
+    TriFingerPlattform(
+        robot_interfaces::TriFingerTypes::Data::Ptr robot_data,
+        trifinger_object_tracking::ObjectTrackerData::Ptr object_tracker_data)
+        : robot_interfaces::TriFingerTypes::Frontend(robot_data),
+          object_tracker_frontend_(object_tracker_data)
+    {
+    }
+
+    // alias
+    robot_interfaces::TriFingerTypes::Observation get_robot_observation(
```

Comment:
As the class is inheriting from the robot frontend, add this alias which is clearer than just `get_observation`.  Unfortunately, the latter will also still be there, so maybe not the cleanest solution.